src/main/java/com/redislabs/BenchmarkRunner.java

```diff
@@ -64,12 +62,9 @@ public static void main(String[] args) {
     public void run() {
         int requestsPerClient = numberRequests / clients;
         int rpsPerClient = rps / clients;
-        GenericObjectPoolConfig poolConfig = new GenericObjectPoolConfig();
-        poolConfig.setMaxTotal(connections);
-        poolConfig.setMaxIdle(connections);
-        JedisPool pool = new JedisPool(poolConfig, hostname,
-                port, timeout, password);
-        RedisGraph rg = new RedisGraph(pool);
+        DefaultJedisClientConfig clientConfig = DefaultJedisClientConfig.builder().socketTimeoutMillis(timeout).connectionTimeoutMillis(timeout).password(password).build();
+        HostAndPort hap = new HostAndPort(hostname,port);
+        UnifiedJedis uredis = new UnifiedJedis( hap, clientConfig);
```

Comment:
It's still possible to set pool config as it was earlier.

```
GenericObjectPoolConfig<Connection> poolConfig = new GenericObjectPoolConfig<>();
...
...
UnifiedJedis uredis = new JedisPooled(poolConfig, hostname, port, timeout, password);
```

Reply:
addressed in the latest commit